product_tier_validation/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "category": "Products",
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "installable": True,
+    "installable": False,
```

Comment:
:thinking:

Reply:
This belong to TMP commit too, would be removed when depend PR merged